proxy/src/keystore.rs

```diff
@@ -0,0 +1,211 @@
+//! Storage of secret keys.
+
+use std::convert::Infallible;
+use std::fmt;
+
+use secstr::{SecStr, SecUtf8};
+
+use librad::keys;
+use librad::paths;
+use radicle_keystore::{
+    crypto::{Pwhash, SecretBoxError},
+    file, FileStorage, Keystore, SecretKeyExt,
+};
+use radicle_registry_client::{ed25519, CryptoError, CryptoPair};
+
+const LIBRAD_KEY: &str = "librad.key";
+const REGISTRY_KEY: &str = "registry.key";
+
+/// Storage for putting and getting the necessary cryptographic keys.
+pub struct Keystorage {
```

Comment:
Uncertain about a bundled `Keystorage`. I envision this to be injected wherever we need keys, but then again most places depend on either of them and seldomly on both together. Do you see downsides to have `keystore::Coco` and `keystore::Registry` instead?

Reply:
I suppose this `Signer` business would cover this too? :thinking: